src/main/resources/META-INF/reaktivity/amqp.idl

```diff
@@ -136,6 +136,7 @@ scope amqp
             AmqpBinary deliveryTag;
             uint32 messageFormat;
             uint8 flags;   // set of AmqpTransferFlag
+            int32 deferred = 0;
```

Comment:
Please put `deferred` first as it is a candidate for promotion to generic `DATA` frame.